apps/frontend/src/store/store.ts

```diff
@@ -0,0 +1,17 @@
+import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
+import previewTreeReducer from './previewTree/previewTreeSlice';
+
+export const store = configureStore({
+    reducer: {
+        previewTree: previewTreeReducer
+    }
+});
+
+export type AppDispatch = typeof store.dispatch;
+export type RootState = ReturnType<typeof store.getState>;
```

Comment:
Why not just `State`, instead of `RootState` ?

Reply:
I copy the docs